src/Relation/Binary/Domain/Structures.agda

```diff
@@ -0,0 +1,66 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Structures for domain theory
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Binary.Domain.Structures where
+
+open import Data.Product using (_×_; _,_)
+open import Data.Nat.Properties using (≤-trans)
+open import Function using (_∘_)
+open import Level using (Level; _⊔_; suc)
+open import Relation.Binary.Bundles using (Poset)
+open import Relation.Binary.Domain.Definitions
+open import Relation.Binary.Morphism.Structures using (IsOrderHomomorphism)
+
+private variable
+  o ℓ e o' ℓ' e' ℓ₂ : Level
+  Ix A B : Set o
```

Comment:
A lot of these variables don't seem to be needed (and indeed are not standard names in the library) while other names used throughout the file (e.g. `c`, `ℓ₁`) could be variables.